src/Vote.jsx

```
@@ -384,14 +404,53 @@
+          <CheckItem label={"Klassenlistennr."} checked={listIndex} />
+          <div className="break" />
+          {extraFields?.map((e, i) => (
+            <>
+              <CheckItem
+                key={i}
+                label={e}
+                checked={extraFieldsValues[i]?.trim()}
+              />
+              <div className="break" />
+            </>
+          ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)